contrib/storage-cassandra/src/test/java/org/apache/drill/exec/store/cassandra/TestCassandraSuit.java

```diff
@@ -65,7 +66,12 @@ public static void tearDownCluster() {
 
   private static void startCassandra() {
     cassandra = new CassandraContainer<>("cassandra")
-        .withInitScript("queries.cql");
+                      .withInitScript("queries.cql")
+                      .withStartupTimeout(Duration.ofMinutes(3))
```

Comment:
It look like 2m is enough. There is no one failure for Cassandra Test Container this value
https://github.com/vdiravka/drill/commit/b5b3a17f19989f4c9214b125fdb4751cba1515ab